flow/connectors/snowflake/client.go

```diff
@@ -82,3 +85,18 @@ func (c *SnowflakeConnector) getTableCounts(tables []string) (int64, error) {
 	}
 	return totalRecords, nil
 }
+
+func snowflakeIdentifierNormalize(identifier string) string {
+	// https://www.alberton.info/dbms_identifiers_and_case_sensitivity.html
+	// Snowflake follows the SQL standard, but Postgres does the opposite.
+	// Ergo, we suffer.
+	if strings.ToLower(identifier) == identifier {
```

Comment:
A `utils.IsLower(string)` / `utils.IsUpper(string)` could be implemented using `unicode` package's `IsLetter` / `IsUpper` / `IsLower`

https://pkg.go.dev/unicode